nuget/PerformanceSensitive.Analyzers/Microsoft.PerformanceSensitive.Analyzers.targets

```diff
@@ -5,8 +5,9 @@
     <PerformanceSensitiveAttributePath Condition="'$(PerformanceSensitiveAttributePath)' == ''">$(MSBuildThisFileDirectory)PerformanceSensitiveAttribute$(DefaultLanguageSourceExtension)</PerformanceSensitiveAttributePath>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(GeneratePerformanceSensitiveAttribute)' == 'true'">
-    <Compile Include="$(PerformanceSensitiveAttributePath)" Condition="Exists($(PerformanceSensitiveAttributePath))" Visible="false" />
+  <ItemGroup Condition="'$(GeneratePerformanceSensitiveAttribute)' == 'true' and Exists($(PerformanceSensitiveAttributePath))">
+    <Compile Include="$(PerformanceSensitiveAttributePath)" Visible="false" />
+    <EmbeddedFiles Include="$(PerformanceSensitiveAttributePath)" />
```

Comment:
Can you add a comment for this? It looks like `EmbeddedResource` but is totally different. 😁

Reply:
Sure